internal/manifest/validate_node_pool.go

```diff
@@ -2,8 +2,14 @@ package manifest
 
 import (
 	"fmt"
+	"regexp"
 
 	"github.com/go-playground/validator/v10"
+	k8sV1 "k8s.io/api/core/v1"
+)
+
+const (
+	labelRegex = "^(?:[a-z0-9A-Z](?:[-_.]?[a-z0-9A-Z]){0,61})?(?:\\.[a-z0-9A-Z](?:[-_.]?[a-z0-9A-Z]){0,61})*(?:\\/[a-z0-9A-Z](?:[-_.]?[a-z0-9A-Z]){0,61})?$"
```

Comment:
This is the regexp to validate is `label` proper? If so, why arent we using `"github.com/go-playground/validator/v10"` for that? https://github.com/go-playground/validator#baked-in-validations

Reply:
I believe the regex is proper, tried it on a few cases where it failed/passed accordingly.

I used this regex as we cannot use the validator, since we need to iterate over keys/values of a map, not just to validate a single string. However, if you know how to use the go validator for a map in this way, let me know :).